torch/quantization/fx/quantize.py

```diff
@@ -583,6 +583,21 @@ def load_arg(a: Argument) -> Argument:
                     if node in copy_nodes:
                         copy_nodes.remove(node)
 
+        # rule 6: remove observers for inputs of masked_fill since
+        # it is boolean Tensor
+        # TODO: we need type info from proxy, what type of Tensor this is
+        # similar to TorchScript
+        if (node.op, node.target) == ("call_method", "masked_fill"):
```

Comment:
could we instead structure this as something like https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/utils.py#L406, 

```
if second_input_arg_is_boolean_tensor(node):
  ...
```

Reply:
sure